### DISCUSSION TITLE
Initial render shuffle animation.

### DISCUSSION THREAD

**Discussion by boldurean** [barvian/number-flow#29] [Example request]
Is it possible to add ability to shuffle value on initial render and animate-in after transition delay is set?

I will try to describe what I try to achieve.
1. Component renders
2. Number flow gets the value.
3. Shuffles numbers until N milliseconds (could be customized) and animates in after.

Thanks for great work!

**Comment by barvian** [comment]
Hi, could you show an example of the shuffling part? I'm not sure what that would look like exactly but I'd imagine it'd be possible by using the `animated` prop on NumberFlow.
